### ISSUE TITLE
properties calling properties of superclass: false no-member

### ISSUE THREAD

**Issue by mhooreman** [PyCQA/pylint#3269]
### Steps to reproduce

1. Make a class defining a property. Inherit from that class and override the property, this last implementation calls the super property, as shown in the code below

1. Run pylint

```python
"""Calling a super property"""

class A:
    """A parent class"""
    @property
    def test(self):
        """A property"""
        return "test"


class B:
    """A child class"""
    @property
    def test(self):
        """Overriding implementation of prop which calls the parent"""
        return A.test.fget(self) + " overriden"


if __name__ == "__main__":
    print(B().test)
```

#### Remark: use case

The child implementation adds "something" to the parent implementation

### Current behavior

`no-member(E1101): test.py:16:15: B.test: Method 'test' has no 'fget' member`

### Expected behavior

No issue

### pylint --version output

- pylint 2.4.2
- astroid 2.3.1
- Python 3.7.4 (default, Jul 17 2019, 14:07:45)
- [GCC 7.4.0]
-

**Comment by PCManticore** [comment]
Thanks for creating the issue. We should be inferring a kind of property node for properties, while we currently infer them as functions. Might not be too hard to support though.

**Comment by PCManticore** [comment]
Thanks for the report, this is fixed in `master` and will be part of the next `astroid` and `pylint` releases.
